src/main/java/io/supertokens/totp/Totp.java

```diff
@@ -386,13 +391,19 @@ public static void verifyCode(TenantIdentifierWithStorage tenantIdentifierWithSt
         // another API call. We will still check the code against the updated set of
         // devices and store it in the used codes table. This behaviour is okay so we
         // can ignore it.
-        checkAndStoreCode(tenantIdentifierWithStorage, main, userId, devices, code);
+        try{
+            checkAndStoreCode(tenantIdentifierWithStorage, main, userId, devices, code);
+        } catch (TotpNotEnabledException e) {
```

Comment:
catch unknown device here and the throw that as a InvalidTotpException exception